webapp/src/Form/Type/AbstractExternalIdEntityType.php

```diff
@@ -24,21 +25,19 @@ public function __construct(protected readonly EventLogService $eventLogService)
      */
     protected function addExternalIdField(FormBuilderInterface $builder, string $entity): void
     {
-        if ($this->eventLogService->externalIdFieldForEntity($entity) !== null) {
-            $builder->add('externalid', TextType::class, [
-                'label' => 'External ID',
-                'required' => false,
-                'empty_data' => '',
-                'constraints' => [
-                    new Regex(
-                        [
-                            'pattern' => DOMJudgeService::EXTERNAL_IDENTIFIER_REGEX,
-                            'message' => 'Only letters, numbers, dashes, underscores and dots are allowed',
-                        ]
-                    ),
-                    new NotBlank(),
-                ]
-            ]);
-        }
+        $builder->add('externalid', TextType::class, [
+            'label' => 'External ID',
+            'help' => 'Leave empty to generate automatically',
```

Comment:
```suggestion
            'help' => 'Leave empty to generate automatically.',
```

Not sure if we do this normally for help.